tests/test_views.py

```diff
@@ -4026,6 +4026,20 @@ def test_external_login_confirm_email_get_with_another_user_logged_in(self):
         assert_in('/logout?service=', res.location)
         assert_in(url, res.location)
 
+    def test_external_login_confirm_email_twice(self):
+        url = self.user.get_confirmation_url(self.user.username, external_id_provider='orcid', destination='dashboard')
+        print(self.user.email_verifications)
+        res = self.app.get(url, auth=self.auth)
+        assert_equal(res.status_code, 302, 'redirects to cas logout')
+        assert_in('/login?service=', res.location)
+
+        self.user.refresh_from_db()
+        assert_equal(self.user.email_verifications, {})
+
+        res = self.app.get(url, auth=self.auth)
+        assert_equal(res.status_code, 302, 'redirects to cas logout')
```

Comment:
Is this a correct comment? It looks like it's redirecting to the logged in dashboard.

Reply:
yeah it's wrong, it's a good old copy and paste error.